src/langs/en.json

```diff
@@ -15,6 +14,50 @@
         "THANKS_FOR_FEEDBACK": "Thank you for your feedback!",
         "SEND": "Send your feedback"
       }
+    },
+    "SETTINGS": {
+      "TITLE": "App Settings",
+      "SAVE_BTN": "Save",
+      "UP": "Move up",
+      "DOWN": "Move down",
+      "ASSISTANTS": {
+        "TITLE": "Assitants Settings",
+        "ID": "Assistant Id",
+        "NAME": "Assistant Name",
+        "DESCRIPTION": "Assistant Description",
+        "IMAGE": "Assistant Image",
+        "URL": "Please enter a working link to your desired image.",
+        "ADD": "Create new Assistant",
+        "CREATE": "Please create at least one assistant."
+      },
+      "CHAT": {
+        "TITLE": "Chat Settings",
+        "DESCRIPTION": "Chat Description",
+        "INSTRUCTIONS": "Participant Instructions",
+        "END": "End Screen Text",
+        "USER": "Participant"
+      },
+      "EXCHANGES": {
+        "TITLE": "Exchanges Settings",
+        "DESCRIPTION": "Exchanges Description",
+        "INSTRUCTIONS": "Chatbot instructions",
+        "CUE": "Initial Cue",
+        "ASSISTANT": "Assistant",
+        "CREATE_ASSISTANT": "Please create at least one assistant.",
+        "FOLLOW_UP_QUESTIONS": "Number of Follow Up Questions",
+        "ON_COMPLETE": "Instruction on Exchange Completion",
+        "ON_COMPLETE_HELPER": "This text will be shown once the given number of follow up questions have been answered. Leave blank for no message.",
+        "DISABLE_HARD_LIMIT": "Auto Dismiss",
+        "HARD_LIMIT_INFO": "If checked, the next exchange will start automatically after the number of follow up questions is reached.",
+        "CREATE": "Please create at least one exchange."
+      }
+    },
+    "START_INTERACTION": "The dialogue lasts about 5 minutes and you'll receive a small reward to thank you for taking part!",
```

Comment:
Is this the default text? This should definitely be modifiable in the app settings.

Reply:
It would have come after the participant instructions by default. I took it out as you can already add or leave it in the participant instructions. The only difference is that it will not be centerd.